.circleci/config.yml

```diff
@@ -22,17 +22,11 @@ orbs:
 
 machine_executor: &machine_executor
   machine:
-    image: ubuntu-1604:201903-01
+    image: ubuntu-2004:202107-02
   environment:
     - BOTO_CONFIG: /dev/null
     # https://support.circleci.com/hc/en-us/articles/360045268074-Build-Fails-with-Too-long-with-no-output-exceeded-10m0s-context-deadline-exceeded-
     - PYTHONUNBUFFERED: 1
-  steps:
```

Comment:
Not sure if this was required or not, so going to try removing and see what happens

Reply:
Hmm the docs build seems to try to use Python 2.7 even though we specify pys=["3"] in the riotfile 🤔 

https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/12271/workflows/b252e6a7-e039-4e9e-9251-7d711da60766/jobs/901274